### ISSUE TITLE
default_auth script not fetching world_address properly

### ISSUE THREAD

**Issue by Jonatan-Chaverri** [dojoengine/dojo-starter#55]
The [default_auth.sh](https://github.com/dojoengine/dojo-starter/blob/main/scripts/default_auth.sh#L7) script is not being able to fetch world address properly. As you can see it is trying to read that info from manifest.json and that file does not contains that data. Maybe we can fetch it from Scarb.toml

Also I think we should update dojo book to include a hint that user should run this script manually after sozo migrate, that is not being said anywhere and its important to do so (otherwise you get permissions errors that you don't know how to fix).

**Comment by Jonatan-Chaverri** [comment]
@kariy (or someone) please let me know what you think, I can take care of this one.

**Comment by ponderingdemocritus** [comment]
Yes! Please tackle this

**Comment by tarrencev** [comment]
I think we can move default auth to the new overlay auth approach cc @lambda-0x

**Comment by itzlambda** [comment]
@Jonatan-Chaverri i the command fetching world address from manifest file is working as expected for me, perhaps you ran `sozo clean` before calling the script?

regarding moving to overlay system for model authorization it will be available from 0.7.0.alpha.1 (currently we are using 0.6.0 in this project) so will open an issue for it

opened: #56

**Comment by Jonatan-Chaverri** [comment]
@lambda-0x the issue was that "jq" was not installed in my system. I think this tools is not included per default with any Unix or linux distributions nor macOS systems. And the script just throw an error indicating that world address was not found. Perhaps adding documentation regarding the tools required will be nice or throwing some more explicit error on the console.

I spend some time trying to figure out why I got 'not writer' error on katana logs before I discover that I needed to manually run this script to grant my address writer access. Then I run into this issue, I just pasted the world address directly in the script and it worked. 

But hopefully with the new auth model this will be resolved, so closing the issue. Thanks.
